source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,66 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take an
+additional options object as the second parameter. Set the ``fullDocument``
+field of the additional options object to ``'updateLookup'`` to receive
+an event that contains the changes to the document as well as the entire
+altered document. Alternatively, you can set ``fullDocument`` to the
+string ``'default'`` to only receive the altered document, but not the
+changes to the document. 
+
+Process the Change Stream Events
+--------------------------------
+
+You can capture events from a change stream instance by using a listener
+function. Call the watch() command to get a Change Stream instance. Add
+your listener function by calling the `EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_
+method on that instance. Set the value of the first parameter to the
+string ``'change'`` and add your :mdn:`callback function
+<Glossary/Callback_function>` as the second parameter.  The callback
+triggers when a change event is emitted, providing the next available
+document. You can specify logic in the callback to process the event
+document when it is received. 
+
+Call the :node-api:`close() </ChangeStream.html#close>` method on the
+change stream instance to stop processing change events. This method
+closes the change stream and frees resources.
+
+Example
+-------
+
+The following example opens a ``Change Stream`` on the ``movies``
+collection. We create a ``changeListener`` variable that is listener
+function to receive and print change events that occur. 
+
+To emit a change event, we want to perform a change to the collection, such as insertion
+with ``insertOne()``. To prevent the ``insertOne()`` from executing
+before ``changeListener`` can register, we create a ``firstTimer``
+variable that uses :mdn:`setTimeout
+<Web/API/WindowOrWorkerGlobalScope/setTimeout>` to wait 1 second. After
+the insertion of the document, the listener should receive this event.
```

Comment:
"insertion of the document" is passive voice. How about:
"When a document is inserted, the listener receives a corresponding event."

Reply:
Actually my sentence is already active voice, whereas the proposed sentence is not. I still struggle with this, but active voice is when the subject of the sentence performs the action (https://www.grammarly.com/blog/active-vs-passive-voice/)

I'm no English prof but broken down, I believe the following is accurate:
 
In my sentence: "After the insertion of the document, the listener should receive this event." 
- the subject is the listener, and the listener receives the event
- the first half of the sentence is not using passive voice, it refers to an event that occurred on an object (the document) that is not the subject of the sentence.

Whereas for the sentence: "When a document is inserted, the listener receives a corresponding event.":
- both the document and the listener are compound subjects. 
-  "the listener receives a corresponding event" is using active voice since the subject ("listener") performs the action (receiving an event).
- "When a document is inserted" is passive voice because the subject (a document) is being acted on by an outside party ("is inserted")

That being said, I do like your sentence generally, but I prefer the way it's currently written, and don't think changing it should block this PR

Reply:
Sorry, you're right that my recommendation is passive voice! I was conflating my recommendation with a sentence containing two independent clauses. My new suggestion is  "When you insert a document, the listener receives the corresponding event". What do you think about that?

The phrase "insertion of the document" sounds awkward, or perhaps the cause-effect is not strongly-associated. Consider a different context with a parallel structure:

Your version: "After the payment of taxes, the government should receive money."

vs

My new suggested version: "When you pay taxes, the government receives the corresponding money."

I prefer the latter, but if you want to stick with the former, that's fine. If I mentioned that this "[blocks] this PR", I didn't mean to as there were several other requested changes.

Reply:
"When you insert a document, the listener receives the corresponding event" sounds fine to me. See above comment